recipes-openxt/initrdscripts/initramfs-xenclient/init.sh

```diff
@@ -27,6 +27,26 @@ ROOT_READONLY=
 DEFINIT=/sbin/init
 FIRSTBOOT_FLAG=/boot/system/firstboot
 
+is_tpm_2_0 () {
+    [ -e /sys/class/tpm/tpm0/device/description ] && cat /sys/class/tpm/tpm0/device/description | grep "2.0" &>/dev/null
```

Comment:
Is this going to be a reliable test for a 2.0 TPM? I have a 1.2 TPM and "description" DNE. Does the description node exist for all 2.0 TPMs.